src/mplhep/plot.py

```diff
@@ -397,12 +476,69 @@ def iterable_not_string(arg):
         if binticks:
             _slice = int(round(float(len(final_bins)) / len(ax.get_xticks()))) + 1
             ax.set_xticks(final_bins[::_slice])
+    elif flow == "show":
+        if binticks:
+            _slice = int(round(float(len(final_bins)) / len(ax.get_xticks()))) + 1
+            ax.set_xticks(final_bins[::_slice])
     else:
         ax.set_xticks(_bin_centers)
         ax.set_xticklabels(xtick_labels)
 
     if x_axes_label:
         ax.set_xlabel(x_axes_label)
+    if flow == "hint" or flow == "show":
+        d = 0.9  # proportion of vertical to horizontal extent of the slanted line
+        trans = mpl.transforms.blended_transform_factory(ax.transData, ax.transAxes)
+        kwargs = dict(
+            marker=[(-0.5, -d), (0.5, d)],
+            markersize=15,
```

Comment:
could the marker size be somehow derived in relation to figure size?

Reply:
yes, it possible, I take height to change the size

```
ax_h = ax.bbox.height
        kwargs = dict(
            marker=[(-0.5, -d), (0.5, d)],
            markersize=ax_h*0.05,
            linestyle="none",
            color="k",
            mec="k",
            mew=1,
            clip_on=False,
            transform=trans,
        )
```